### PR TITLE
Header

### PR DESCRIPTION
**Pull request by stevenGarciaDev**
* Implement navbar to be used in App.js
* Add default styling for box sizing and color
* Add section headings for home and search page
* Update title for page
* Add integration test for navbar header

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import App from './App';
 
-test('renders App', () => {
-  render(<App />);
+const navLinkCases = [['Search'], ['How it works'], ['About']];
+
+describe('Header', () => {
```
Comment (jkettmann) [inline src/App.test.js:8]: Nice tests :+1:

File: src/App.test.js [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+  it('App logo link apears on the page', () => {
+    expect(
+      screen.getByRole('link', { name: /logo/i }),
+    ).toBeInTheDocument();
+  });
+
+  test.each(navLinkCases)(
+    '"%s" link points to the correct page',
+    (navLinkText) => {
+      const textRegex = new RegExp(navLinkText, 'i');
```
Comment (jkettmann) [inline src/App.test.js:24]: Do you need a regexp here?

File: src/App.test.js [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+    );
+  });
+
+  it('App logo link apears on the page', () => {
+    expect(
+      screen.getByRole('link', { name: /logo/i }),
+    ).toBeInTheDocument();
+  });
+
+  test.each(navLinkCases)(
```
Comment (jkettmann) [inline src/App.test.js:21]: I'd probably not use a const here and move the array directly here. It's a bit harder to follow what the test is doing if you have `navLinkCases` at the top of the file

Comment (stevenGarciaDev) [reply src/App.test.js:21]: Sounds good, I will work on refactoring it.

**Pull request merged**